packages/web-react/src/components/Header/demo/HeaderInvertedWithActions.tsx

```diff
@@ -1,29 +1,31 @@
 import React, { useState } from 'react';
-import { ComponentStory } from '@storybook/react';
-import { Link } from '../../Link';
 import { Button } from '../../Button';
+import { Link } from '../../Link';
+import {
+  Header,
+  HeaderDesktopActions,
+  HeaderDialog,
+  HeaderDialogActions,
+  HeaderDialogCloseButton,
+  HeaderDialogLink,
+  HeaderDialogNav,
+  HeaderDialogNavItem,
+  HeaderLink,
+  HeaderMobileActions,
+  HeaderNav,
+  HeaderNavItem,
+} from '..';
 import SpiritLogo from './SpiritLogo';
-import Header from '../Header';
-import HeaderMobileActions from '../HeaderMobileActions';
-import HeaderDesktopActions from '../HeaderDesktopActions';
-import HeaderNav from '../HeaderNav';
-import HeaderNavItem from '../HeaderNavItem';
-import HeaderLink from '../HeaderLink';
-import HeaderDialog from '../HeaderDialog';
-import HeaderDialogCloseButton from '../HeaderDialogCloseButton';
-import HeaderDialogActions from '../HeaderDialogActions';
-import HeaderDialogNav from '../HeaderDialogNav';
-import HeaderDialogNavItem from '../HeaderDialogNavItem';
-import HeaderDialogLink from '../HeaderDialogLink';
 
-const Story: ComponentStory<typeof Header> = () => {
+const HeaderInvertedWithActions = () => {
   const [isOpen, setOpen] = useState(false);
 
   const handleOpen = () => setOpen(true);
   const handleClose = () => setOpen(false);
 
   return (
     <>
+      <p>Using the Offcanvas JS plugin.</p>
```

Comment:
This is React, it does not need any JS plugin.

Reply:
@adamkudrna @crishpeen I think we should remove these subheadings also from web and web-twig.

Reply:
We agreed that we remove this subheading in all demos. So please remove this one. Thanks.

Reply:
removed